app/registries/foreman/plugin.rb

```diff
@@ -317,6 +317,24 @@ def tests_to_skip(hash)
       end
     end
 
+    # Adds setting definition
+    #
+    # ===== Example
+    #
+    #   settings do |mapper|
+    #     mapper.category(:cfgmgmt, N_('Configuration Management')) do |cat_mapper|
+    #       cat_mapper.setting(:use_cooler_puppet, default: true,
+    #                                              description: N_('This will ensure we install new and shiny puppet on Host instead of rusty one'),
```

Comment:
```suggestion
    #                                              description: N_('This will ensure we install a new and shiny Puppet on hosts instead of a rusty one'),
```